Assets/Scripts/UI/LeaderBoardManager.cs

```diff
@@ -35,6 +35,10 @@ public class LeaderBoardManager : MonoBehaviour
     public string m_FilePrefix = "PowerpuffBuns";
     private string m_FilePath;
 
+    [Header("Unlock Character")]
```

Comment:
unlock 되었을때 짜잔 하고 나오는 그림인거죠?
```suggestion
    [Header("Image to be shown when character gets unlocked")]
```